install/sessions.csv

```diff
@@ -217,7 +220,8 @@ KOKO,GeoNet,GNS,TRIMBLE NETR9,GPS+GLO,30s,0,geonet,trimble_netr9 x4,2015-12-08T0
 KOKO,GeoNet,GNS,TRIMBLE ALLOY,GPS+GLO,30s,0,geonet,trimble_netr9 x4,2024-12-08T23:37:02Z,9999-01-01T00:00:00Z
 KORO,GeoNet,GNS,TRIMBLE NETRS,GPS,30s,4,geonet,trimble_5700 x4,2007-06-14T23:55:00Z,2015-09-10T00:00:00Z
 KORO,GeoNet,GNS,TRIMBLE NETR9,GPS+GLO,30s,0,geonet,trimble_netr9 x4,2015-09-10T00:00:01Z,2024-07-25T03:32:31Z
-KORO,GeoNet,GNS,TRIMBLE ALLOY,GPS+GLO,30s,0,geonet,trimble_netr9 x4,2024-07-25T03:32:32Z,9999-01-01T00:00:00Z
+KORO,GeoNet,GNS,TRIMBLE ALLOY,GPS+GLO,30s,0,geonet,trimble_netr9 x4,2024-07-25T03:32:32Z,2025-02-19T02:28:43Z
+KORO,GeoNet,GNS,TRIMBLE ALLOY,GPS+GLO+GAL+BDS+QZSS,30s,0,geonet,trimble_netr9 x4,2025-02-19T02:28:44Z,9999-01-01T00:00:00Z
```

Comment:
The receiver has not been reconfigurated, so this changes should be excluded from the PR until it is configured properly.